accumulo/src/main/java/com/yahoo/ycsb/db/accumulo/AccumuloClient.java

```diff
@@ -100,62 +104,72 @@ public void init() throws DBException {
   @Override
   public void cleanup() throws DBException {
     try {
-      if (bw != null) {
-        bw.close();
+      Iterator<BatchWriter> iterator = writers.values().iterator();
+      while (iterator.hasNext()) {
+        BatchWriter writer = iterator.next();
+        writer.close();
```

Comment:
Can't this throw MutationsRejectedException? Should we safeguard this and remove it from the writers anyway? Will it matter at this point in the process?